src/components/Board.test.tsx

```diff
@@ -0,0 +1,86 @@
+import Board from './Board';
+import setup from '../userEvent';
+import { getAllByRole, getRoles, screen } from '@testing-library/dom';
```

Comment:
suggestion: 使っていない変数は削除していただけると

```suggestion
import { screen } from '@testing-library/dom';
```